runtime/acala/src/weights/module_cdp_treasury.rs

```diff
@@ -48,16 +48,16 @@ use sp_std::marker::PhantomData;
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> module_cdp_treasury::WeightInfo for WeightInfo<T> {
 	fn auction_collateral() -> Weight {
-		(66_209_000 as Weight)
+		(2_908_193_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(6 as Weight))
-			.saturating_add(T::DbWeight::get().writes(7 as Weight))
+			.saturating_add(T::DbWeight::get().writes(304 as Weight))
 	}
```

Comment:
We need to adjust `MaxAuctionsCount` so the worst case scenario has less writes, currently a value of 100 in runtimes. Alternatively we could pass max_auction_count as a argument for `auction_collateral` and the weight be a function of that.